src/components/PostDetails/PostDetails.jsx

```diff
@@ -1,45 +1,99 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
 import { NewCommentForm } from '../NewCommentForm';
+import { getPostComments, removePostComment } from '../../api/comments';
+import { GetPostDetails } from '../../api/posts';
 import './PostDetails.scss';
 
-export const PostDetails = () => (
-  <div className="PostDetails">
-    <h2>Post details:</h2>
-
-    <section className="PostDetails__post">
-      <p>sunt aut facere repellat provident occaecati excepturi optio</p>
-    </section>
-
-    <section className="PostDetails__comments">
-      <button type="button" className="button">Hide 2 comments</button>
-
-      <ul className="PostDetails__list">
-        <li className="PostDetails__list-item">
-          <button
-            type="button"
-            className="PostDetails__remove-button button"
-          >
-            X
-          </button>
-          <p>My first comment</p>
-        </li>
-
-        <li className="PostDetails__list-item">
-          <button
-            type="button"
-            className="PostDetails__remove-button button"
-          >
-            X
-          </button>
-          <p>sad sds dfsadf asdf asdf</p>
-        </li>
-      </ul>
-    </section>
-
-    <section>
-      <div className="PostDetails__form-wrapper">
-        <NewCommentForm />
-      </div>
-    </section>
-  </div>
-);
+export const PostDetails = ({ selectedPostId }) => {
+  const [details, setDetails] = useState(null);
+  const [comments, setComments] = useState([]);
+  const [hiddenComments, setHiddenComments] = useState(true);
+
+  useEffect(() => {
+    GetPostDetails(selectedPostId)
+      .then(setDetails);
+
+    getPostComments(selectedPostId)
+      .then(setComments);
+  }, [selectedPostId]);
+
+  const removeComment = (commentId) => {
+    setComments(comments.filter(comment => comment.id !== commentId));
+    removePostComment(commentId);
+  };
+
+  const addComment = (commentvef) => {
+    setComments(state => [...state, commentvef]);
```

Comment:
```suggestion
    setComments(state => [...state, commentvef]);
```
commentvef що це значить?